CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+# Contributing to Flutter Templates
+
+First off, thanks for taking the time to contribute! 🎉
+
+Read our [Code of Conduct](./CODE_OF_CONDUCT.md) to keep our community approachable and respectable.
+
+Please make sure to read the relevant section before making your contribution. It will make it a lot easier for us maintainers and smooth out the experience for all involved. The community looks forward to your contributions.
```

Comment:
```suggestion
Please make sure to read the relevant section before making your contribution. It will make it much easier for us maintainers and smooth out the experience for all involved. The community looks forward to your contributions.
```